### PR TITLE
CASMINST-6048

### PR DESCRIPTION
**Pull request by phalseth-hpe**
# Description

    This change adds a procedure and related script to manually add the embedded repo to Nexus.
The procedure and script were verified to be working properly on Fanta.

https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-6048

# Checklist

- [x ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x ] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by mtupitsyn — APPROVED** [review]

**Review by rustydb — CHANGES_REQUESTED** [review]
Some feedback.

I have more incoming.

Why does the script and yaml need to live in here?

File: operations/package_repository_management/Nexus_Import_Embedded_Repo.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+(`ncn-m#`)
+
+```bash
+/usr/share/doc/csm/scripts/operations/nexus/setup-embedded-repository.sh $CSM_RELEASE $PATH_TO_ROOT_OF_EXTRACTED_CSM_TARBALL_CONTENT
+```
+
+On the success, the above command will report:
+
+\+ Nexus setup complete
```
Comment (rustydb) [inline operations/package_repository_management/Nexus_Import_Embedded_Repo.md:29]: Wrap this in triple backticks, since it is script output.

**Review by rustydb — APPROVED** [review]
Just a nit left.

**Comment by rustydb** [comment]
Shell check is failing, please fix.

File: scripts/operations/nexus/setup-embedded-repository.sh [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+# OTHER DEALINGS IN THE SOFTWARE.
+
+set -exo pipefail
+
+#two required inputs
+#1 RELEASE_VERSION
+#2 ROOTDIR of extracted CSM tarball
+
+#RELEASE_VERSION
+RELEASE_VERSION="$1"
```
Comment (rustydb) [inline scripts/operations/nexus/setup-embedded-repository.sh:31]: nit: align this to CSM_RELEASE as well.

```bash
CSM_RELAESE=${1:-$CSM_RELEASE}
```

Comment (phalseth-hpe) [reply scripts/operations/nexus/setup-embedded-repository.sh]: done

**Review by heemstra — APPROVED** [review]

**Pull request merged**